### ISSUE TITLE
[BUG]: Nothing is scrapped

### ISSUE THREAD

**Issue by riz-ahmed** [pystardust/ytfzf#661]
### Describe the bug
apparently the program doesn't fetch for any query. I have build it directly from the source. When I request for a query such as ytfzf -t richard stallman, the prompt seems to be trasferred to a sub-process from which it never returns (i tried waiting upto 1 hour). After which, when I decided to CTRL-D the process, I receive the following message on the prompt

![image](https://user-images.githubusercontent.com/38962235/223840302-2be5d969-649f-417a-a8d7-30e1d9871f39.png)

### Information
 - OS: Mac OS 12.0.1 (Intel)
 - Terminal: tired both with iTerm2 as well as Alacritty
 - Ytfzf version: 2.5.4
- Output of `ls -l "$(which sh)"` (if you're using fish: `ls -l (which sh)`): -rwxr-xr-x  1 root  wheel  121008 Oct 18  2021 /bin/sh
 - (if is a thumbnail issue) run `ytfzf --thumbnail-log=log.txt` and post the file:

**Comment by Euro20179** [comment]
try doing `ytfzf --rii <search>`

**Comment by riz-ahmed** [comment]
![image](https://user-images.githubusercontent.com/38962235/223944243-0f60a18a-ca79-4561-a9c8-8c7b2b2aab73.png)

it doesn't seem to go any far this time too. The prompt never returns to the main process unless i hit CTRL-C to just cancel it. Any other ideas? thanks!

**Comment by Euro20179** [comment]
Sorry, I dont have other ideas

**Comment by riz-ahmed** [comment]
thanks for the tip, doesn't seem to still work on Mac OS. It just works fine though on Linux

**Comment by HanoJing** [comment]
> thanks for the tip, doesn't seem to still work on Mac OS. It just works fine though on Linux

Oh interesting, do you have fzf installed on MacOS?

**Comment by riz-ahmed** [comment]
yes, I have fzf installed which is working fine for various other console programs. Tired this both with Intel and M1 chips, just works on Linux

**Comment by HanoJing** [comment]
Okay, interesting, then there might be something wrong with python itself or it working differently on MacOS.
I'm out of ideas, what else could be the issue, sorry.

**Comment by tabletseeker** [comment]
Updated and working ytfzf project here:
https://github.com/tabletseeker/ytfzf_prime
